### PR TITLE
Symbolize crash backtraces using the Abseil debugging library

### PR DESCRIPTION
**Pull request by jason-simmons**
See https://github.com/flutter/flutter/issues/83286

### CODE REVIEW

File: fml/paths.h [hunk 1]
```diff
@@ -13,6 +13,7 @@
 namespace fml {
 namespace paths {
 
+std::pair<bool, std::string> GetExecutablePath();
```
Comment (chinmaygarde) [inline fml/paths.h:16]: I think (in a later patch maybe) we should refactor these to use std::optional now that it is available to us.

Comment (chinmaygarde) [reply fml/paths.h:16]: Can we dry this up a little bit by putting `GetExecutableDirectoryPath` in the common implementation (It'll just call `GetDirectoryName(GetExecutablePath())` after an optional value check) and having platform specific implementations of `GetExecutablePath()`?

File: fml/platform/win/paths_win.cc [hunk 2]
```diff
@@ -63,7 +63,15 @@ std::pair<bool, std::string> GetExecutableDirectoryPath() {
   if (read_size == 0 || read_size == MAX_PATH) {
     return {false, ""};
   }
-  return {true, GetDirectoryName(std::string{path, read_size})};
+  return {true, std::string{path, read_size}};
+}
+
+std::pair<bool, std::string> GetExecutableDirectoryPath() {
```
Comment (chinmaygarde) [inline fml/platform/win/paths_win.cc:69]: See the comment about moving this into `fml/paths.cc`.

**Review by chinmaygarde — APPROVED** [review]

File: fml/paths.h (hunk 1 above)
Comment (jason-simmons) [reply fml/paths.h:16]: done

**Review by chinmaygarde — APPROVED** [review]

**Pull request merged**